build/scripts/version-update/after-release/after-release.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+func main() {
```

Comment:
rather than the heavy duplication between `after-release` and `before-release`, I would recommend using a flag. For a script like this, no worries using the standard flag library: https://pkg.go.dev/flag

Examples in our repo include: https://github.com/googleforgames/agones/blob/main/examples/simple-game-server/main.go#L41-L49

Reply:
I have updated the code to make it a single script using Flag. Thanks!